app/Kconfig

```diff
@@ -561,6 +561,10 @@ endmenu
 
 if SETTINGS
 
+config ZMK_SETTINGS_RESET_ON_START
+    bool "Delete all persistent settings when the keyboard boots"
+    default n
```

Comment:
As we fixed up recently, this line is unnecessary, and even can cause issues (see https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults):

```suggestion
```